package.json

```diff
@@ -9,7 +9,8 @@
   "type": "module",
   "scripts": {
     "postinstall": "pnpm compile",
-    "compile": "pnpm -r -w=false run compile",
+    "compile": "pnpm -r -w=false compile",
```

Comment:
I know this was here before, but what is `-w` supposed to be? According to the docs this does not exist and I would assume for a boolean option to be `false` by default.

Reply:
It's the very first option in the docs?
https://pnpm.io/pnpm-cli#-w---workspace-root

Reply:
Hmm after removing it I do not see any difference in the output, so you might be onto something